libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Conway/Instances/TxBody.hs

```diff
@@ -58,15 +60,10 @@ type ShelleyTxBodyTypes era =
    , Maybe TxAuxDataHash
    ]
 
-instance
-  ( EraTxOut era
-  , EncCBOR (TxCert era)
-  ) =>
-  HasSimpleRep (ShelleyTxBody era)
-  where
-  type SimpleRep (ShelleyTxBody era) = SOP '["ShelleyTxBody" ::: ShelleyTxBodyTypes era]
+instance HasSimpleRep (TxBody ShelleyEra) where
+  type SimpleRep (TxBody ShelleyEra) = SOP '["ShelleyTxBody" ::: ShelleyTxBodyTypes ShelleyEra]
```

Comment:
I imagine these types: `ShelleyTxBodyTypes`, `AlonzoTxBodyTypes` etc could also be simplified to remove the era parameter